plugins/clnrest/README.md

```diff
@@ -0,0 +1,37 @@
+# CLNRest
+
+CLNRest is a lightweight Python-based core lightning plugin that transforms RPC calls into a REST service. By generating REST API endpoints, it enables the execution of Core Lightning's RPC methods behind the scenes and provides responses in JSON format.
+
+## Installation
+
+Install required packages with `pip install json5 flask flask_restx gunicorn pyln-client` or `pip install -r requirements.txt`.
+
+## Configuration
+
+- --rest-protocol: Specifies the REST server protocol. Default is HTTPS.
+- --rest-host: Defines the REST server host. Default is 127.0.0.1.
+- --rest-port: Sets the REST server port to listen to. Default is 3010.
+- --rest-certs: Defines the path for HTTPS cert & key. Default path is same as RPC file path to utilize gRPC's client certificate. If it is missing at the configured location, new identity (`client.pem` and `client-key.pem`) will be generated.
+
+## Plugin
+
+- It can be configured by adding `plugin=/<path>/clnrest/clnrest.py` to the Core Lightning's config file.
+
+## Server
+
+With the default configurations, the Swagger user interface will be available at https://127.0.0.1:3010/. The POST method requires `rune` and `nodeid` headers for authorization.
```

Comment:
can you add an example cURL command that shows how this is used?

Reply:
iirc there is a Open API page that generate the code for you here

Reply:
@niftynei Added `cURL` section in README with some sample commands. @vincenzopalazzo yes, Open API (Swagger) also generates cURL command but sometimes people prefer to use it directly. So adding the direct section might be helpful for those users.